internal/service/project/resource_project_test.go

```diff
@@ -610,16 +612,17 @@ func TestAccProjectGov_withProjectOwner(t *testing.T) {
 	)
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheckGov(t) },
```

Comment:
if gov already configured so we could enable the tests in CI?

Reply:
I think we can but I believe we don't have a workflow for gov right now so we can do this in a separate ticket.
Including @andreaangiolillo in case he has anything to add regarding CI testing on gov